services/galley/test/integration/API.hs

```diff
@@ -455,8 +454,9 @@ postCryptoMessageVerifyRejectMissingClientAndRepondMissingPrekeysJson = do
       assertMismatchWithMessage (Just "client mismatch") [(eve, Set.singleton ec)] [] []
   let x = responseJsonUnsafeWithMsg "ClientMismatch" r1
   -- Fetch all missing clients prekeys
+  b <- view tsUnversionedBrig
   r2 <-
-    post (b . zUser alice . path "/users/prekeys" . json (missingClients x))
+    post (b . zUser alice . path "v1/users/prekeys" . json (missingClients x))
```

Comment:
This looks strange: using `b` as an unversioned brig and then followed by a path prefixed with "v1". Is that a mistake?

Reply:
Why would it be a mistake? I don't understand why you think this and the other cases below are strange.

Reply:
Tests pass and that suggests it works, but I find it strange to read: it is unversioned and at version v1. Maybe if this was hidden away in a utility function (such as `apiVersion` that takes a version as input), then I wouldn't have problems with it.

Reply:
Sure, we can add an `apiVersion`, but I don't quite understand what's strange about it. To make a versioned path, you start with the unversioned one and you add a version prefix to it. It's just the normal thing to do :shrug:.

Reply:
I agree, this is how it is achieved under the hood, i.e., these are the steps to construct a versioned path. I was just trying to say I'd prefer it put in a utility function (where we care about details of how to construct a valid versioned path) compared to exposing this detail at every use site to avoid always reading it's unversioned first and then we make it versioned.